docs/en/observability/aws-serverless-transform.asciidoc

```diff
@@ -0,0 +1,416 @@
+:aws: AWS
+
+[[aws-serverless-transform]]
+= Transform AWS data

```

Comment:
These are all things you need to do in that config.yaml before you upload it to an s3 bucket. It seems to me that all the config-related info should be in the sample place rather than buried under a category like "transform".

Reply:
Maybe the Configure and Transform split isn't the best...have added a short note to the Deploy topic for now while we work out how to integrate/restructure the additional configure and transform details...